sunpy/util/scraper.py

```diff
@@ -152,24 +152,20 @@ def _date_floor(date, timestep):
         date_parts[-1] = date_parts[-1] % 60
         date = datetime(*date_parts)
         orig_time_tup = date.timetuple()
-        if timestep == relativedelta(seconds=1):
```

Comment:
note that I deleted this branch, it's now the default. Do you think this is an issue?

Reply:
Yea that will still work